conversion-measurement/functions/apps/adtech.js

```diff
@@ -18,24 +18,11 @@ const functions = require('firebase-functions')
 const express = require('express')
 const cookieParser = require('cookie-parser')
 require('dotenv').config({ path: `.env.${process.env.NODE_ENV}` })
-const structuredHeaders = require('structured-headers')
```

Comment:
Does this mean we can remove the dependency on `structured-headers` from `conversion-measurement/functions/package.json`?